client/my-sites/importer/importer-logo.jsx

```diff
@@ -19,34 +19,37 @@ import SocialLogo from 'components/social-logo';
  */
 import './importer-logo.scss';
 
-const ImporterLogo = ( { icon } ) => {
+const ImporterLogo = ( { icon, className = 'importer__service-icon' } ) => {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+	if ( 'blogger' === icon ) {
+		icon = 'blogger-alt';
+	}
+
 	if ( includes( [ 'wordpress', 'blogger-alt', 'squarespace' ], icon ) ) {
-		return <SocialLogo className="importer__service-icon" icon={ icon } size={ 48 } />;
+		return <SocialLogo className={ className } icon={ icon } size={ 48 } />;
 	}
 
 	if ( 'wix' === icon ) {
-		return <WixLogo />;
+		return <WixLogo className={ className } />;
 	}
 
 	if ( 'godaddy-gocentral' === icon ) {
-		return <GoDaddyGoCentralLogo size={ 48 } />;
+		return <GoDaddyGoCentralLogo size={ 48 } className={ className } />;
```

Comment:
Why is GoDaddy the only one that has a size specified?

Reply:
I'm not sure why a size param was implemented in this particular icon but not in the others – I've left it in and given it a default rather than lose the extra flexibility it brings.